backend/utils/sql/models.py

```diff
@@ -12,21 +13,26 @@ class JobPosting:
     job_type: str
     source_url: str
     publish_date: str
+    candidate_required_location: str
+
 
 @dataclass
 class ModelData:
     id: int
-    job_title: str
+    candidate_required_location: str
```

Comment:
I'm wondering if it is appropriate to include a candidate_required_location of the model rather than pre-filtering from the model any potential locations. I'm not opposed, but also want to make sure that each time we send records to the model, we have a minimally scoped set of records for vetting by the model.

Reply:
Sorry I'm a bit confused about what you mean here, would you rather me remove candidate_required_location from the ModelData portion? Would the model strictly remove all jobs that don't match a particular location or would it just rank them lower?

Reply:
Because we are using intrinsically remote jobs, I was thinking that the recommendation engine should just use descriptions, categories and tags to do the matching. I was thinking of removing it for that, but could be persuaded otherwise.

Reply:
okay sure, just removed it from the model data